src/src/PluginStructs/P016_data_struct.cpp

```diff
@@ -63,7 +63,7 @@ void P016_data_struct::ExecuteCode(uint32_t  Code) {
   uint32_t _now = millis();
```

Comment:
is `_now` still needed?

Reply:
It is used to set the `iLastCmdTime`, a few lines below, that could indeed be a direct assignment of `millis()`.